rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportController.java

```diff
@@ -76,15 +67,15 @@ public RepositoryLinksResource process(RepositoryLinksResource resource) {
             description = "Generate the reports.",
             tags = {"Reports"}
     )
-    @GetMapping(value = REPORTS_URL)
+    @GetMapping(value = SW360Constants.REPORTS_URL)
     public void getProjectReport(
             @Parameter(description = "Projects with linked releases.")
             @RequestParam(value = "withlinkedreleases", required = false, defaultValue = "false") boolean withLinkedReleases,
             @Parameter(description = "Report download format.", schema = @Schema(allowableValues = {"xls", "xlsx"}))
             @RequestParam(value = "mimetype", required = false, defaultValue = "xlsx") String mimeType,
             @Parameter(description = "Project id.")
             @RequestParam(value = "projectId", required = false) String projectId,
-            @Parameter(description = "Module name.", schema = @Schema(allowableValues = {PROJECTS, COMPONENTS}))
+            @Parameter(description = "Module name.", schema = @Schema(allowableValues = {SW360Constants.PROJECTS, SW360Constants.COMPONENTS}))
```

Comment:
Please update the `allowableValues` to also include `LICENSES` and `PROJECT_RELEASE_SPREADSHEET_WITH_ECCINFO`